Assets/AppCenter/Plugins/AppCenterSDK/Analytics/Blank/WrapperTransmissionTargetInternal.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+//
+// Licensed under the MIT license.
+
+#if (!UNITY_IOS && !UNITY_ANDROID && !UNITY_WSA_10_0) || UNITY_EDITOR
+using Microsoft.AppCenter.Unity.Analytics.Internal;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.AppCenter.Unity.Analytics
+{
+
+    public class WrapperTransmissionTargetInternal
+    {
+
+        public static void TrackEvent(IntPtr transmissionTarget, string eventName)
```

Comment:
shouldn't we use `RawType` here instead of `IntPtr` and define it as
```
#if UNITY_IOS
    using RawType = System.IntPtr;
#elif UNITY_ANDROID
    using RawType = UnityEngine.AndroidJavaObject;
#else
    using RawType = System.Object;
#endif
```
?